plugin/core/protocol.py

```diff
@@ -212,6 +212,11 @@ class InsertTextMode:
     'items': List[CompletionItem],
 }, total=True)
 
+Hover = TypedDict('Hover', {
+    'contents': Union[str, Dict[str, str], list],
```

Comment:
`List[str]` is not entirely correct. A more "correct" representation would be `List[Union[str, dict]]` (because `MarkedString` can also be a `dict`)

reference from spec:
```
export interface Hover {
	contents: MarkedString | MarkedString[] | MarkupContent;
	range?: Range;
}

type MarkedString = string | { language: string; value: string };

export interface MarkupContent {
	kind: MarkupKind;
	value: string;
}
```

I wanted to create types for `MarkedString` and `MarkupContent` but that change would affect other files... so I do not want to do it in this PR. I want to keep this PR concise and just fix the issue for LSP-tailwindcss :)